### PR TITLE
Create redistimeseries-1.8-release-notes.md

### PR DESCRIPTION
**Pull request by LiorKogan**

### CODE REVIEW

**Review by rrelledge — CHANGES_REQUESTED** [review]

File: content/modules/redistimeseries/release-notes/redistimeseries-1.8-release-notes.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+Details:
+
+- Bug fixes:
+
+    - [#1360](https://github.com/RedisTimeSeries/RedisTimeSeries/pull/1360) Potential crash when upgrading from v1.6 to 1.8 if there are compactions with `min` or `max` aggregation (MOD-4559)
+    - [#1370](https://github.com/RedisTimeSeries/RedisTimeSeries/issues/1370) Potential crash when using [TS.REVRANGE](https://redis.io/commands/ts.revrange/) or [TS.MREVRANGE](https://redis.io/commands/ts.mrevrange/) with aggregation
+    - [#1347](https://github.com/RedisTimeSeries/RedisTimeSeries/pull/1347) When adding samples with [TS.ADD](https://redis.io/commands/ts.add/) or [TS.MADD](https://redis.io/commands/ts.madd/) using  `*` as timestamp, the timestamp could differ between master and replica shards
+
+- Improvements:
+    - [#1215](https://github.com/RedisTimeSeries/RedisTimeSeries/pull/1347) OSS cluster: Support TLS and IPv6; introduce new configuration parameter: [OSS_GLOBAL_PASSWORD](https://redis.io/docs/stack/timeseries/configuration/#oss_global_password)
```
Comment (rrelledge) [inline content/modules/redistimeseries/release-notes/redistimeseries-1.8-release-notes.md:45]: The PR # doesn't match the link.

File: content/modules/redistimeseries/release-notes/redistimeseries-1.8-release-notes.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+* **New reducers**
+
+  Till version 1.8, only the `sum`, `min`, and `max` could be used as reducer types.
+
+  It is now possible, for example, to calculate the maximal temperature per timeframe for each sensor and then report the average temperature (avg reducer) over groups of sensors (grouped by a given label's value).
+
+  This can be specified with the new reducer types ([TS.MRANGE](https://redis.io/commands/ts.mrange/) and [TS.MREVRANGE](https://redis.io/commands/ts.mrevrange/)): `avg`, `range`, `count`, `std.p`, `std.s`, `var.p`, and `var.s`.
+
+{{<note>}}
+RDB version 7 since 1.8.2. RDB files are not backward compatible.
```
Comment (rrelledge) [inline content/modules/redistimeseries/release-notes/redistimeseries-1.8-release-notes.md:143]: Is there a missing word in the first sentence?

File: content/modules/redistimeseries/release-notes/redistimeseries-1.8-release-notes.md (hunk 1 above)
Comment (LiorKogan) [reply content/modules/redistimeseries/release-notes/redistimeseries-1.8-release-notes.md]: Fixed. Thank you.

File: content/modules/redistimeseries/release-notes/redistimeseries-1.8-release-notes.md (hunk 2 above)
Comment (LiorKogan) [reply content/modules/redistimeseries/release-notes/redistimeseries-1.8-release-notes.md]: Fixed. Now similar to what we have in v1.6.16

**Review by rrelledge — APPROVED** [review]

**Pull request merged**